### PR TITLE
got barometer working on new refactored system

### PR DESCRIPTION
**Pull request by superjax**
This does not use the base sensor class we were talking about, but it does work.  Until we have the base sensor class, I figured we can use this.

### CODE REVIEW

File: include/mavlink/v1.0 [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 6251d3f235d653e88eec8f599787adfbd361f358
+Subproject commit 124514055a9f0da425dea2c887e5ec14dd9d859f
```
Comment (superjax) [inline include/mavlink/v1.0:1]: This is a new mavlink commit (it includes the `SMALL_BARO` message)

**Comment by dpkoch** [comment]
@superjax Oh, I just saw this. I refactored the sensor stuff kind of like we talked about (not a base class since the number/type of values for a sensor varies, but a collection of convenience classes. See #23). I'll pull this in for the new format and create a new pull request.

**Comment by superjax** [comment]
That's alright.  Thanks for doing the time synchronization stuff.  I actually have an IMU calibration routine I'll throw into the `imu::correct` fuction

**Comment by dpkoch** [comment]
So I wonder if we want to do IMU calibration on the ROS side. Since we're using IMU onboard, it seems like any calibration that needs to happen should happen onboard, and we shouldn't have to redo it on the ROS side. Thoughts?

**Comment by superjax** [comment]
I am just afraid that the naze won't be able to do the least-squares required for calibration

**Comment by superjax** [comment]
Well, what if we provide a 'good' temperature compensation, but the ROS utility can be used to fine-tune it for the performance junkies?

**Pull request closed without merging**